ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/datacolumns/util/DataColumnSidecarELManagerImplTest.java

```diff
@@ -521,14 +554,31 @@ public void shouldStopRetry_whenElBlobsAreFetched() {
     verify(executionLayer, times(2)).engineGetBlobAndCellProofsList(any(), any());
 
     assertThat(asyncRunner.hasDelayedActions()).isFalse();
-    verify(dataColumnSidecarPublisher)
-        .accept(
-            argThat(dataColumnSidecars -> dataColumnSidecars.size() == sampleGroupCount),
-            argThat(origin -> origin == RemoteOrigin.LOCAL_EL));
+
+    verifyRecovery(true);
+  }
+
+  private void verifyNoRecovery() {
```

Comment:
If you could come to other word than recovery for EL retrieved blobs, it would be good. If not, let it go as is. I'm lost on ideas.